executor/new_distsql.go

```diff
@@ -41,6 +41,13 @@ type DataReader interface {
 	doRequestForDatums(datums [][]types.Datum, goCtx goctx.Context) error
 }
 
+func handleIsExtra(col *expression.Column) bool {
+	if col != nil && col.ID == model.ExtraHandleID {
```

Comment:
return col != nil && col.ID == model.ExtraHandleID
Add comment for the function.